opnode/rollup/derive/l1_block_info_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"math/rand"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
```

Comment:
We should be fuzzing the following:
- That `Marshall/Unmarshall` never panic
- That `Marshall` and `Unmarshall` match abigen

Reply:
Going to add fuzzing as a new issue. This bugfix is important and has been stuck for long enough. We already have your e2e test to verify the minimum works.